cwf/gateway/tools/uesim_cli/main.go

```diff
@@ -246,44 +256,52 @@ func getConfiguredSubscribers() ([]*protos.UEConfig, error) {
 		return nil, fmt.Errorf("unable to convert %T to map %v", subscribers, rawMap)
 	}
 	var ues []*protos.UEConfig
-	for k, v := range rawMap {
-		imsi, ok := k.(string)
+	if len(params) > 0 {
```

Comment:
I think it would be easier if you populate your own `RawMap map[interface{}]interface{}` when an IMSI is provided.  The code below is duplication of what it is after line 279

Reply:
changed  the name rawMap to subscriberCfg to avoid confusion with the earlier declared rawMap.

Reply:
Maybe I am missing something, but the code at lines 261-280 and code 283--300 look the same

Can't not you just add the parameters to `rawMap` when a single IMSI is configured and then skip all the code from 261-280?